autoload/multiple_cursors.vim

```diff
@@ -1239,7 +1236,10 @@ function! s:wait_for_user_input(mode)
         exec 'let s:char = "'.substitute(char_mapping, '<', '\\<', 'g').'"'
         break
       endif
-      sleep 1m
+      if s:get_time_in_ms() > (s_time + &timeoutlen)
+        break
+      endif
+      sleep 100m
```

Comment:
I suppose this wouldn't work very well for people with `timeoutlen` smaller than 100ms, but that seems uncommon? Maybe `10m` would be safer still though?